.github/workflows/python.yml

```
@@ -0,0 +1,27 @@
+# Python unit tests
+name: Python
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches: ['master']
  pull_request:
    branches: ['master']
```
Otherwise, the build will always be run twice for one PR.